onecodex/viz/_metadata.py

```diff
@@ -116,16 +120,19 @@ def plot_metadata(
         import altair as alt
         import pandas as pd
 
-        if rank is None:
-            raise OneCodexException("Please specify a rank or 'auto' to choose automatically")
+        if len(self._classifications) == 0:
+            raise PlottingException(f"too few samples: n={len(self._classifications)}")
```

Comment:
I think this is covered below? Let's use the error message from below either way since it's user-facing in Custom Plots